packages/cursorless-engine/src/processTargets/targets/DestinationImpl.ts

```diff
@@ -31,12 +33,21 @@ export class DestinationImpl implements Destination {
       indentationString ?? this.isLineDelimiter
         ? getIndentationString(target.editor, target.contentRange)
         : "";
-    this.insertionPrefix =
-      target.prefixRange != null
-        ? target.editor.document.getText(target.prefixRange)
+  }
+
+  // TODO: we can't have "async/await" in a constructor so we initialize elsewhere
```

Comment:
@pokey do we like this way to call async functions when constructing an object?

Reply:
I wonder if we could get the insertion prefix lazily when we actually need it?